controllers/funding/materials/views/materials.njk

```diff
@@ -2,7 +2,7 @@
 {% from "components/icons.njk" import iconClose %}
 {% from "./materials-fields.njk" import textInput, radios with context %}
 
-{% extends "controllers/common/views/cms-page.njk" %}
+{% extends "../../../common/views/cms-page.njk" %}
```

Comment:
Again, auto refactoring weirdness. The previous path works fine due to how nunkucks resolves paths.

Reply:
>nunkucks
![image](https://user-images.githubusercontent.com/394376/78912875-0ebea880-7a80-11ea-9c41-d0bee3112e2b.png)